iroh/src/discovery/pkarr/dht.rs

```diff
@@ -90,57 +81,29 @@ struct Inner {
 }
 
 impl Inner {
-    async fn resolve_relay(&self, key: pkarr::PublicKey) -> Option<Result<DiscoveryItem>> {
-        tracing::info!("resolving {} from relay {:?}", key.to_z32(), self.relay_url);
-
-        let maybe_packet = self
-            .pkarr_relay
-            .as_ref()
-            .expect("checked")
-            .resolve(&key)
-            .await;
-        match maybe_packet {
-            Ok(Some(signed_packet)) => match NodeInfo::from_pkarr_signed_packet(&signed_packet) {
-                Ok(node_info) => {
-                    tracing::info!("discovered node info from relay {:?}", node_info);
-                    Some(Ok(DiscoveryItem::new(node_info, "relay", None)))
-                }
-                Err(_err) => {
-                    tracing::debug!("failed to parse signed packet as node info");
-                    None
-                }
-            },
-            Ok(None) => {
-                tracing::debug!("no signed packet found in relay");
-                None
-            }
-            Err(err) => {
-                tracing::debug!("failed to get signed packet from relay: {}", err);
-                Some(Err(err.into()))
-            }
-        }
-    }
-    async fn resolve_dht(&self, key: pkarr::PublicKey) -> Option<Result<DiscoveryItem>> {
-        tracing::info!("resolving {} from DHT", key.to_z32());
+    async fn resolve_pkarr(&self, key: pkarr::PublicKey) -> Option<Result<DiscoveryItem>> {
+        tracing::info!(
+            "resolving {} from relay and DHT {:?}",
+            key.to_z32(),
+            self.relay_url
+        );
 
         let maybe_packet = self.pkarr.resolve(&key).await;
         match maybe_packet {
-            Ok(Some(signed_packet)) => match NodeInfo::from_pkarr_signed_packet(&signed_packet) {
+            Some(signed_packet) => match NodeInfo::from_pkarr_signed_packet(&signed_packet) {
                 Ok(node_info) => {
-                    tracing::info!("discovered node info from DHT {:?}", &node_info);
-                    Some(Ok(DiscoveryItem::new(node_info, "mainline", None)))
+                    tracing::info!("discovered node info {:?}", node_info);
+                    Some(Ok(DiscoveryItem::new(node_info, "dht", None)))
```

Comment:
is this always `dht` or should we rename this to `pkarr`?

Reply:
Good point, it supports both relay and DHT, so changed to `pkarr`.